src/test/java/HomeWork16.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class HomeWork16 extends BaseTest {
+    @Test
+    public void registrationNavigation() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+
+        WebElement registrationButton= driver.findElement(By.cssSelector("[id='hel']"));
+        registrationButton.click();
+
+        String registrationUrl = "https://qa.koel.app/register";
```

Comment:
Provided link should be : `String registrationUrl = "https://qa.koel.app/registration.php";`